examples/create_libraries/example/hw_example.dart

```diff
@@ -0,0 +1,5 @@
+import 'package:create_libraries/hw_mp.dart';
+
+main() {
+  print('greeting: ${message}');
```

Comment:
Or (see below):
```dart
main(List<String> arguments) {
  if (arguments.isEmpty) {
     alarm();
  } else {
     alarm(arguments.first);  
  }
}
```

Reply:
I ended up not using this code, to keep the focus on the import.